src/bench/base58.cpp

```diff
@@ -13,7 +13,7 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
```

Comment:
If we're going to change this for pedantic correctness (which I'm not arguing for), I'd 100x rather make this a ```std::array<unsigned char, 32>``` and iterate properly.